src/components/Articles.jsx

```diff
@@ -1,21 +1,43 @@
 import React, { useEffect,useState,useContext } from "react"
+import {useParams} from "react-router-dom"
 import axios from 'axios'
+import { getArticles } from "../utils/api"
 
 const Articles = () => {
+   
     const[articles, setArticles] = useState([])
-}
+
 
 useEffect(() => {
-    axios.get("")
-    .then(({res})=> {
-       return res.json();
-    })
-    .then((data) => {
-        setArticles(data.articles)
-    })
-})
+   getArticles().then((articles) => {
+   setArticles(articles)
+   })
+},[])
+
 
 
     return (
-        <div className="body">
-                    </div> )
\ No newline at end of file
+       <div>
+        <h1>Articles</h1>
+       {articles.map((article) => {
+        return (
+            <div key={article.article_id}> 
+            <h2>
+             {article.title}
+            <p>{article.topic}</p>
+            <p>{article.author}</p>
+            <p> {article.body}</p>
+            <p>{article.created_at}</p>
+            <p> {article.votes}</p>
```

Comment:
To make it slightly more readable it might be good to use little icons to make it clear which of these bit of data relate to what, you can use https://react-icons.github.io/react-icons/ for this, but this isn't mandatory - just a slight point of improvement 🙂